File: consulting/src/main/kotlin/it/unibo/lss/fcla/consulting/application/persistence/ConsultingRepository.java
package it.unibo.lss.fcla.consulting.application.persistence;

import it.unibo.lss.fcla.consulting.common.EventSourcedRepository;
import it.unibo.lss.fcla.consulting.domain.consulting.Consulting;

/**
 * @author Stefano Braggion
 */
public class ConsultingRepository extends EventSourcedRepository<Consulting> {
    public ConsultingRepository(EventStore eventStore) {
        super(eventStore);
    }
}

File: consulting/src/main/kotlin/it/unibo/lss/fcla/consulting/application/persistence/ConsultingRepository.kt
package it.unibo.lss.fcla.consulting.application.persistence

import it.unibo.lss.fcla.consulting.common.EventSourcedRepository
import it.unibo.lss.fcla.consulting.domain.consulting.Consulting
import it.unibo.lss.fcla.consulting.domain.contracts.IEventStore

class ConsultingRepository(eventStore: EventStore) : EventSourcedRepository<Consulting>(eventStore)